source/mesh_deformation/fastscape.cc

```diff
@@ -522,10 +522,48 @@ namespace aspect
                                              &sand_transport_coefficient,
                                              &silt_transport_coefficient);
 
+          // generate a combined array for kf kd both onshore and offshore
+          std::vector<double> combined_kd(fastscape_array_size);
+          std::vector<double> combined_kf(fastscape_array_size);
+          for (unsigned int i = 0; i < fastscape_array_size; ++i)
+            {
+              combined_kf[i] = bedrock_river_incision_rate_array[i];
+              combined_kd[i] = bedrock_transport_coefficient_array[i];
+              // for cells below sea level, grep the marine sediment data
+              if (elevation[i] < current_sea_level)
+                {
+                  combined_kf[i] = sediment_river_incision_rate;
+                  combined_kd[i] = sediment_transport_coefficient;
+                };
```

Comment:
No need to overwrite an array just written into:
```suggestion
              if (elevation[i] >= current_sea_level)
              {
              combined_kf[i] = bedrock_river_incision_rate_array[i];
              combined_kd[i] = bedrock_transport_coefficient_array[i];
              }
              // for cells below sea level, grep the marine sediment data
              else
                {
                  combined_kf[i] = sediment_river_incision_rate;
                  combined_kd[i] = sediment_transport_coefficient;
                };
```